plugins/out_stackdriver/stackdriver.h

```diff
@@ -65,6 +67,14 @@ struct flb_stackdriver {
     flb_sds_t zone;
     flb_sds_t instance_id;
     flb_sds_t instance_name;
+    flb_sds_t cluster_name;
```

Comment:
I don't think these fields should be here right after instance_name. They are not GCP specific. They might be another section as 'kubernetes specific'

Reply:
updated